docs/articles/nunit/release-notes/framework.md

```diff
@@ -4,6 +4,73 @@ uid: frameworkreleasenotes
 
 # Framework Release
 
+## NUnit 4.0.0-alpha.1 - TBA
+
+This is the first alpha release of version 4 of NUnit. This is a gradual improvement of the version 3, and can be used with the same runners as before, although a minor upgrade might be needed for some.  For the NUnit3TestAdapter, you will need version 4.5 to run tests with NUnit 4. This release have more than 50 fixes and improvements implemented.
+
+The minimum target framework supported is now dotnet framework 4.6.2, and .net core 3.1.
```

Comment:
```suggestion
The minimum target framework supported is now dotnet framework 4.6.2, and .net core 3.1.
```

❓ The release notes PR says .NET 6.0 rather than .NET Core 3.1 -- we should ensure they are in sync, whichever decision we go with.